src/index/formatFileStructure/toFractalTree/createAddDependency.ts

```diff
@@ -0,0 +1,7 @@
+export function createAddDependency(dependencyIndex: Record<string, string[]>) {
+  return (fileName: string, dependency: string) => {
+    const is = fileName in dependencyIndex;
+    if (!is) dependencyIndex[fileName] = [];
+    dependencyIndex[fileName].push(dependency);
+  };
+}
```

Comment:
A module with 7 LOC.

Reply:
Understandable.I'll put it into an existing file